docs/adding_a_new_gravity_endpoint_into_metaphysics.md

```diff
@@ -0,0 +1,78 @@
+## Adding a gravity endpoint into Metaphysics
+
+This doc is aimed at helping backend devs or beginner front end devs on how to add an existing gravity endpoint into Metaphysics.
+
+This article assumes that you already have your local machine setup and have your graphiql.app or other app (Postman, Insomnia etc) setup. If you need help with that, you can start [here](https://github.com/artsy/metaphysics#setting-up-your-local-graphiql)
+
+### Steps to add an existing endpoint
+
+1. Find an existing PR or commit which adds something similar to what you want. If you are looking for a PUT endpoint, then [this](https://github.com/artsy/metaphysics/commit/987b21f501bb3483d196720d4a252c25b704352e) is a great example.
+1. Follow the structure of the PR to add your own endpoint and input/outputs. Some further tips can be found below.
+1. After the code has been written, you can run `yarn dump:local` in the Metaphysics directory to generate the new schema files. This script (and other useful scripts) can be found in the `package.json` file.
+1. Start your local server with `yarn start`
+1. With your favourite graphql app, you should be able to hit the local server to refresh the schema. In insomnia, you can click on the "schema" button in the GraphQL panel to manually refresh this, if you don't have auto-refresh turned on (also found in this list)
+1. You should be able to hit your local metaphysics, which will hit staging gravity by default.
```

Comment:
Having `local server` in the previous point and `local metaphysics` in this one is a bit confusing (since they actually mean the same thing). 

Maybe this point can just read "During development MP should be pointing at Gravity staging (unless otherwise configured in `.env.shared`)

Reply:
Replaced all instances with `local Metaphysics server` and also updated the point you mentioned!